### PR TITLE
chore: replace in-publish package with specific script

### PR DESCRIPTION
**Pull request by JaKXz**
Hi! Thank you so much for this package! I thought I'd return some love and noticed a couple maintainery things I could help with.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,8 @@
   "scripts": {
     "clean": "rimraf dist es tmp lib",
     "test": "jest",
-    "prepublish": "in-publish && npm run build || not-in-publish",
+    "prepublishOnly": "npm run build",
+    "prebuild": "npm run clean",
```
Comment (jamesplease) [inline package.json]: It looks like we’ll be cleanin’ twice with this, since `npm run build` also runs `clean`?

**Comment by jamesplease** [comment]
Thank you @JaKXz ! This is great. I had one little comment. What do you think?

File: package.json (hunk 1 above)
Comment (JaKXz) [reply package.json]: Whoops I meant to take that out of build. Will fix!

**Comment by jamesplease** [comment]
The build was failing due to what I believe was a breaking change in some dependency of this project. I went ahead and fixed the problem in #54 , then rebased this branch off of master.

Thanks again for the great PR, @JaKXz !

**Comment by jamesplease** [comment]
Alright, everything looks good. Merge time! :v:

**Pull request merged**